<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>661</width>
     <height>361</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000000</width>
     <height>1000000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;맑은 고딕&quot;;
color: rgb(32, 178, 170);
border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;
background-color: #FFFFFF;</string>
   </property>
   <property name="title">
    <string>접속정보</string>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
      <horstretch>13</horstretch>
      <verstretch>7</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="connMsgEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;맑은 고딕&quot;;
color: rgb(32, 178, 170);
border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;맑은 고딕&quot;;
color: rgb(32, 178, 170);
border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;맑은 고딕&quot;;
color: rgb(32, 178, 170);
border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;</string>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QListWidget" name="clientListView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>256</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;맑은 고딕&quot;;
color: rgb(32, 178, 170);
border-style: solid;
border-width: 2px;
border-color: #20B2AA;
border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>101</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;맑은 고딕&quot;;
color: rgb(32, 178, 170);</string>
    </property>
    <property name="text">
     <string>고객 접속리스트</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../QT/bokclient (2)/bokclient/bokimage/iogo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
